code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -302,7 +302,10 @@ namespace <#= qualifiedNamespace #>
                             return;
                         }
 
-                        entityManager.AddComponentData(entity, new AuthorityLossImminent<<#= componentDetails.TypeName #>>());
+                        entityManager.AddComponentData(entity, new AuthorityLossImminent<<#= componentDetails.TypeName #>> {
+                            AuthorityLossAcknowledged = false,
+                            AuthorityLossAcknowledgmentSent = false
```

Comment:
As @jamiebrynes7 said, I'm in favour of not having more than one component for this. Ideally if you want to specify some transient other information I'd add another tag component, like `AuthorityLossAcked`. That way you can also predicate an inject group on it. I find it odd when you have a mix of components that we expect users to write to and us to write to and if a user writes to the wrong one it won't do what they expect.